package.json

```diff
@@ -28,11 +27,13 @@
     "test": "npm run lint && webpack && mocha",
     "acquire": "node test/util/acquire.js",
     "build": "npx webpack",
-    "build-debug": "npx webpack --mode none"
+    "build-debug": "npx webpack --mode none",
+    "prepublishOnly": "npm run build"
   },
   "repository": "git://github.com/w3c/webidl2.js",
   "main": "dist/webidl2.js",
-  "module": "index.js",
+  "exports": "./index.js",
```

Comment:
Note that this makes it no longer possible to import anything other than `index.js`:
- <https://nodejs.org/api/packages.html#packages_package_entry_points>
- <https://nodejs.org/api/packages.html#packages_conditional_exports>
- <https://nodejs.org/api/packages.html#packages_exports>

Reply:
Good point, and I think we never considered internal modules as stable, so still a good change I guess. Or were you using them?

Reply:
It makes `dist/webidl2.js` unimportable in newer **Node** versions.

---

It should probably also be renamed to `dist/webidl2.cjs`, or add a `dist/package.json` file with:
```json
{ "type": "commonjs" }
```

Reply:
Why should it be allowed to be imported in newer Node when there is `index.js`?